captum/optim/_core/optimization.py

```diff
@@ -76,17 +75,16 @@ def loss(self) -> torch.Tensor:
             - **loss** (*tensor*):
                         Size of the tensor corresponds to the targets passed.
         """
-        image = (
-            self.input_param()._t[None, ...]
-            if self.input_param()._t.dim() == 3
-            else self.input_param()._t
+        input_t = (
+            self.input_param()[None, ...]
+            if self.input_param().dim() == 3
+            else self.input_param()

```

Comment:
I think that `input_t` can just be equal to `self.input_param()`, as a 4d shape isn't technically required.